data/battle_anim_scripts.s

```diff
@@ -382,40 +382,40 @@ gBattleAnims_StatusConditions:: @ 81C76F8
 	.4byte StatusCondition_Nightmare
 
 	.align 2
-gBattleAnims_Unknown1:: @ 81C771C
-	.4byte Unknown_81D63B4
-	.4byte Unknown_81D63EA
-	.4byte Unknown_81D63F3
-	.4byte Unknown_81D6430
-	.4byte Unknown_81D6438
-	.4byte Unknown_81D647E
-	.4byte Unknown_81D6489
-	.4byte Unknown_81D661C
-	.4byte Unknown_81D6690
-	.4byte Unknown_81D676E
-	.4byte Unknown_Weather_Rain
-	.4byte Unknown_Weather_HarshSunlight
-	.4byte Unknown_Weather_Sandstorm
-	.4byte Unknown_Weather_Hail
-	.4byte Unknown_81D6813
-	.4byte Unknown_81D6821
-	.4byte Unknown_81D6852
-	.4byte Unknown_81D6877
-	.4byte Unknown_81D68D5
-	.4byte Unknown_81D6934
-	.4byte Unknown_81D69F9
-	.4byte Unknown_81D6A39
-	.4byte Unknown_81D6A7C
+gBattleAnims_General:: @ 81C771C
+	.4byte General_CastformChange
```

Comment:
I feel these should have a "battle animation" prefix, rather than just "General" or "Special".  Maybe "BattleAnimGeneral".

Reply:
Why? They are local labels, so they only exist in this file.